docs/maps/clean-data.asciidoc

```diff
@@ -0,0 +1,183 @@
+[role="xpack"]
+[[maps-clean-your-data]]
+=== Clean your data
+
+// https://github.com/elastic/kibana/issues/135319
+
+Geospatial fields in {es} have certain restrictions that need to be addressed before upload. On this section a few recipes will be presented to help troubleshooting common issues on this type of data.
+
+[float]
+==== Convert to GeoJSON or Shapefile
+
+With https://gdal.org/programs/ogr2ogr.html[ogr2ogr] (part of the https://gdal.org[GDAL/OGR] suite) it is pretty straight forward to convert datasets from dozens of formats into a GeoJSON or Esri Shapefile. For example, converting a GPX file can be achieved with the following commands:
+
+[source,sh]
+----
+# Example GPX file from https://www.topografix.com/gpx_sample_files.asp
+#
+# Convert the GPX waypoints layer into a GeoJSON file
+$ ogr2ogr \
+  -f GeoJSON "waypoints.geo.json" \ # Output format and file name
+  "fells_loop.gpx" \ # Input File Name
+  "waypoints" # Input Layer (usually same as file name)
+
+# Extract the routes layer into a GeoJSON file
+$ ogr2ogr -f "GeoJSON" "routes.geo.json" "fells_loop.gpx" "routes"
+----
+
+[float]
+==== Set up the correct coordinate reference system (CRS)
+
+{es} only supports WGS84 Coordinate Reference System. Also with `ogr2ogr`, converting from one coordinate system to WGS84 is usually supported but it depends on the source CRS.
```

Comment:
How about "{es} only supports WGS84 Coordinate Reference System. Use `ogr2ogr` to convert from one coordinate system to WGS84."